### PR TITLE
Release notes for ipsec-client plugin version 3.6.0

### PR DESCRIPTION
**Pull request by agrawalkaushik**
DOCS-382 #time 1h

### CODE REVIEW

File: docs/plugin_ipsec_client.md [hunk 1]
```diff
@@ -256,6 +258,37 @@ exit
 
 The `<remote-address-1>` and `<remote-address-2>` represent the `node > ipsec-client > remote > host` address. The `ipsec` tenant is allowed as per the tenant defined under `ipsec-client > tenant`. The user can split the tunnels into separate services per the routing requirements.
 
+#### Session Record
+
+| Release  | Modification                         |
+| -------- | ------------------------------------ |
+| 3.6.0    | Support for leveraging session-record was introduced |
+
+Various features introduced as part of the [3.6.0](#release-360) version require the user to enable session records on the egress tunnel services such as `ipsec-tunnel` shown above. The following sample config shows the recommend profile that can be configured and associated with the egress tunnel service.
```
Comment (Chr1st0ph3rTurn3r) [inline docs/plugin_ipsec_client.md:267]: The following sample config shows the **recommended**

File: docs/plugin_ipsec_client.md [hunk 2]
```diff
@@ -770,6 +804,36 @@ exit
...
+
+**Release Date:** Oct 13, 2023
+
+**Router Version** 128T-ipsec-2.4.0-4
+
+#### New Features and Improvements
+
+- **WAN-2815** Improve the memory and cpu consumption of the IPsec plugin on the router
+
+For the ipsec plugin, several individual processes were consolidated into a single controller process providing an overall reduction in CPU and memory when the plugin is enabled on a SSR.
```
Comment (Chr1st0ph3rTurn3r) [inline docs/plugin_ipsec_client.md:817]:
```suggestion
For the ipsec plugin, several individual processes were consolidated into a single controller process providing an overall reduction in CPU and memory when the plugin is enabled on **an** SSR.
```

File: docs/plugin_ipsec_client.md [hunk 3]
```diff
@@ -770,6 +804,36 @@ exit
...
+
+- **I95-50410** SSR VPN to Azure interface not working due to a stuck session
+
+The new version adds support for automatically detecting egress tunnel sessions that are not able to communicate with the IPsec remote server anymore. The recovery happens in form of automatically deleting those session allowing new egress tunnel sessions to be formed. Please refer to the [session-record](#session-record) section on how to enable this new feature.
```
Comment (Chr1st0ph3rTurn3r) [inline docs/plugin_ipsec_client.md:821]:
```suggestion
The new version adds support for automatically detecting egress tunnel sessions that are not able to communicate with the IPsec remote server anymore. The recovery happens in **the** form of automatically deleting those session allowing new egress tunnel sessions to be formed. Please refer to the [session-record](#session-record) section on how to enable this new feature.
```

**Review by Chr1st0ph3rTurn3r — CHANGES_REQUESTED** [review]

File: docs/plugin_ipsec_client.md (hunk 3 above)
Comment (BenMatase) [inline docs/plugin_ipsec_client.md:821]: Those sessions

File: docs/plugin_ipsec_client.md [hunk 4]
```diff
@@ -770,6 +804,36 @@ exit
...
+
+- **WAN-2361** Include the egress wan interface being used for ipsec tunnel for better tunnel insights
```
Comment (BenMatase) [inline docs/plugin_ipsec_client.md:823]: Ipsec tunnels

**Review by BenMatase — CHANGES_REQUESTED** [review]

File: docs/plugin_ipsec_client.md (hunk 3 above)
Comment (BenMatase) [reply docs/plugin_ipsec_client.md:821]: `those session` -> `those sessions` just grammar

Comment (agrawalkaushik) [reply docs/plugin_ipsec_client.md:821]: oh - ah got it!

**Review by MichaelBaj — DISMISSED** [review]

**Review by Chr1st0ph3rTurn3r — DISMISSED** [review]

**Review by BenMatase — APPROVED** [review]

**Review by Chr1st0ph3rTurn3r — APPROVED** [review]

**Pull request merged**